scanner/src/funTest/assets/file-counter-expected-output-for-analyzer-result.yml

```diff
@@ -196,10 +196,8 @@ scanner:
           errors:
           - timestamp: "1970-01-01T00:00:00Z"
             source: "FileCounter"
-            message: "DownloadException: No source artifact URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'."
-          - timestamp: "1970-01-01T00:00:00Z"
-            source: "FileCounter"
-            message: "DownloadException: No VCS URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'."
+            message: "\"DownloadException: No source artifact URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'.\"\
+              , caused by \"DownloadException: No VCS URL provided for 'Gradle:com.here.ort.gradle.example:lib:1.0.0'.\""
```

Comment:
This now needs to be adjusted to "\nCaused by: ", or?

Reply:
Yes, but I expect more tests to fail, so I wanted to wait for the CI results.